### PR TITLE
add: placeholder for TWN population

### PR DESCRIPTION
**Pull request by ccerv1**

### CODE REVIEW

File: figs/data/oss_impact/oss_impact.ipynb [hunk 1]
```diff
@@ -81,7 +81,66 @@
...
+       "      <th>Country</th>\n",
+       "      <th>Alpha-2 code</th>\n",
+       "      <th>Alpha-3 code</th>\n",
+       "      <th>Numeric</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>217</th>\n",
+       "      <td>Taiwan (Province of China)</td>\n",
```
Comment (GregCTWang) [inline figs/data/oss_impact/oss_impact.ipynb:118]: Why add the "(Province of China)" after Taiwan?

File: figs/data/oss_impact/oss_impact.ipynb [hunk 2]
```diff
@@ -81,7 +81,66 @@
...
+       "      <td>TWN</td>\n",
+       "      <td>158</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                        Country Alpha-2 code Alpha-3 code  Numeric\n",
+       "217  Taiwan (Province of China)           TW          TWN      158"
```
Comment (GregCTWang) [inline figs/data/oss_impact/oss_impact.ipynb:129]: Same here

**Review by GregCTWang — CHANGES_REQUESTED** [review]
"Province of China"

File: figs/data/oss_impact/oss_impact.ipynb (hunk 2 above)
Comment (ccerv1) [reply figs/data/oss_impact/oss_impact.ipynb:129]: Yeah, I noticed that... I am simply using public data sources (in this case [IBAN](https://www.iban.com/country-codes)) to translate between 2 letter codes (which GitHub used) and the 3 letter codes that the World Bank population data uses. 

In any case, the figure doesn't contain any labels -- only outlines on a map

File: figs/data/oss_impact/oss_impact.ipynb (hunk 1 above)
Comment (ccerv1) [reply figs/data/oss_impact/oss_impact.ipynb:118]: I didn't add that explicitly --  I just ran a preview of the dataset to confirm the other datasets (except the World Bank) had a row for Taiwan. Once someone updates the data point, then I will re-run and this won't be visible anymore

**Comment by GlenWeyl** [comment]
@ccerv1 Did you get what you need from @GregCTWang?

**Comment by GregCTWang** [comment]
@ccerv1 @GlenWeyl Population by Age 15-64: 16,299,548 (Latest data of Feb 2024) from Dept. of Household Registration Affairs

**Comment by GlenWeyl** [comment]
You have until noon eastern tomorrow to merge!


___________________________________________________

Research Lead, Plural Technology Collaboratory<https://aka.ms/plural>, Research Special Projects

___________________________________________________

Collaborate<https://github.com/pluralitybook/plurality> with Digital Minister Audrey Tang, myself and dozens around the world to tell the story of Plurality<https://www.plurality.net/>



________________________________

**Comment by GregCTWang** [comment]
[Interior Stats Report.xls](https://github.com/pluralitybook/plurality/files/14631214/Interior.Stats.Report.xls)
@ccerv1 we can add this original data file to the read.me later.

**Comment by GlenWeyl** [comment]
By tomorrow I mean today (eastern time)...11 hours.


___________________________________________________

Research Lead, Plural Technology Collaboratory<https://aka.ms/plural>, Research Special Projects

___________________________________________________

Collaborate<https://github.com/pluralitybook/plurality> with Digital Minister Audrey Tang, myself and dozens around the world to tell the story of Plurality<https://www.plurality.net/>



________________________________

**Comment by ccerv1** [comment]
Thanks @GregCTWang! New figure looks much better :)

Will re-run the script and merge it with the new images shortly.


image

**Review by GlenWeyl — APPROVED** [review]
Great stuff!

**Pull request merged**